### ISSUE TITLE
To Start Game - have buttons that majority of players need to make one color

### ISSUE THREAD

**Issue by wadek2** [ExtraLudic/DaedalusLabrynthV2#35]
Each player is presented with three colored buttons. Red Green and Grey. The team needs to get a majority of the buttons a single color.

We need to think through how this would work because its not just everyon on the Slack team, since Sam and I might need to be there to facilitate. Perhaps we name the facilitator the same in all teams and that person is ignored for this?

**Comment by esalling23** [comment]
would the color they need to get be random or just any color 50% of the buttons their choice which one? 
also i think there should be 5 buttons just to decrease the chance of them getting 50% a certain color by luck 
@wadek2 @CivicSam  thoughts?

**Comment by CivicSam** [comment]
They start Red/Green/Gray. Each player sees three buttons, meaning there are 15 buttons total.

In order to start the game at least 8 of the 15 buttons need to be the same color (so three players could turn them all the same, 4 players could make 2 the same, or what have you).

But as you say, it's not that hard to luck into it.

Player one: I made all the buttons red (there are now 7 red buttons)

Player 2: I clicked a button.

Game starts!

I think this starting idea is actually problematic and it has not been paper prototypes. I don't want @esalling23 to build it until we have given it a little more thought.
